tutorials/hana-cloud-alerts-autopilot/hana-cloud-alerts-autopilot.md

```diff
@@ -17,11 +17,11 @@ primary_tag: software-product>sap-hana-cloud
   - How to create a catalog, an input, and a custom command with multiple executors that will request a storage resize of an SAP HANA database instance
   - How to trigger an SAP Automation Pilot command from the Alert Notification Service
 
-An SAP HANA Cloud database or a data lake Relational Engine instance have a set of built-in alerts that when triggered, are sent to the SAP Alert Notification Service (ANS).  This service in addition to being able to forward details of the alert to various channels (email, Microsoft Teams, Slack, etc.), can trigger an [SAP Automation Pilot](https://discovery-center.cloud.sap/serviceCatalog/automation-pilot?region=all) command.  In this tutorial, a command will be created to attempt to resolve a disk use alert.
+An SAP HANA Cloud database or a data lake Relational Engine instance have a set of built-in alerts that when triggered, are sent to the SAP Alert Notification Service (ANS).  This service, in addition to being able to forward details of the alert to various channels (email, Microsoft Teams, Slack, etc.), can trigger an [SAP Automation Pilot](https://discovery-center.cloud.sap/serviceCatalog/automation-pilot?region=all) command.  In this tutorial, a command will be created to attempt to resolve a disk use alert.
```

Comment:
A SAP HANA Cloud database...
This service..., can trigger a ... command.

You only use "an" when the following word starts with a vowel

Reply:
Will make those changes throughout.